docs/container-images.md

```diff
@@ -188,19 +213,271 @@ use. There are some requirements on this image though:
    where your code is going to run in the cloud
 3. The entrypoint must be set to a script which executes
    `/usr/bin/python3 -m sematic.resolvers.worker "$@"`
-4. `/usr/bin/python3` must be a valid python interpreter in the image
+4. `/usr/bin/python3` must be a valid Python interpreter in the image
 5. The home directory inside the image must be writable
 6. The `/sematic/bin/` directory inside the image must be writable
 7. `/sematic/bin` must be on your `PATH`
 
-## Working with images
+#### Multiple base images with Bazel
+
+Please see [Support for multiple images](./multiple-base-images.md).
+
+### Docker
```

Comment:
It might be helpful to discuss and provide an example of a `requirements.txt`-only option. This might be attractive to people who are intimidated by docker. You can of course explain what assumptions are made in this case and what happens under-the-hood.

Reply:
Done.